api/command.go

```diff
@@ -30,6 +30,16 @@ func (c Command) FromGuild() bool {
 	return c.GuildID == nil
 }
 
+// ToCreate return the CommandCreate for this Command
+func (c *Command) ToCreate() *CommandCreate {
```

Comment:
What exactly is this for?

Reply:
I actually use it here <https://github.com/DisgoOrg/disgommand/blob/5b1a6023218f3fe946bdb3628514f28c07249a24/command.go#L29>

Reply:
Ah, for getting the struct ready to create the command 👍